src/Microsoft.AspNetCore.Razor.Language/RazorProjectEngineBuilderExtensions.cs

```diff
@@ -4,11 +4,75 @@
 using System;
```

Comment:
Ported the `RazorEngineBuilderExtension` pieces to `RazorProjectEngineBuilderExtensions`. Only thing about these I changed was that they operate on `RazorProjectEngineBuilder` and not `IRazorEngineBuilder`.